server/src/lint.ts

```diff
@@ -16,9 +16,18 @@ export default async function lint(
     return JSON.parse(result)
   } catch (errors) {
     const errorMessage = "prisma-fmt error'd during linting.\n"
+
     if (onError) {
-      onError(errorMessage + errors)
+      if ((errors as string) && errors.includes('ParsingError')) {
+        // recognized Prisma 1 schema
+        onError(
+          "This file is a GraphQL file and is not supported by the Prisma extension. Please install the GraphQL extension ('https://marketplace.visualstudio.com/items?itemName=Prisma.vscode-graphql') from the marketplace and change the file extension to '.graphql'.",
```

Comment:
I think we should ask @nikolasburk for a better worded instruction here. The file can also be a Prisma 1 schema. Probably you also want to tell them to rename the file extension first, then about the extension so they do not miss the second bit.

Reply:
I would suggest the following:

```
You are currently viewing a Prisma 1 datamodel which is based on GraphQL syntax. The current Prisma VS Code extension doesn't support this syntax. 

To get proper syntax highlighting for this file, please change its file extension to `.graphql` and download the GraphQL VS Code extension: https://marketplace.visualstudio.com/items?itemName=Prisma.vscode-graphql

Learn more here: https://pris.ly/prisma1-vscode
```

I guess we can't use Markdown here?! But in case, it's possible, I'd adjust it as follows:

~~~
You are currently viewing a Prisma 1 datamodel which is based on GraphQL syntax. The current Prisma VS Code extension doesn't support this syntax. 

To get proper syntax highlighting for this file, please change its file extension to `.graphql` and download the [GraphQL VS Code extension](https://marketplace.visualstudio.com/items?itemName=Prisma.vscode-graphql).

Learn more [here](https://pris.ly/prisma1-vscode).
~~~

Reply:
Markdown is not possible here unfortunately. Also multiple lines are not allowed, i.e. they get removed automatically (https://github.com/Microsoft/vscode/blob/8616dbae8bc2abf7972a45449b0fb6b2b2d0f429/src/vs/workbench/common/notifications.ts#L412-L413).

Reply:
Actually it is possible to use Markdown, had a typo earlier so I thought this just wouldn't be possible :sweat_smile: I changed that now.
Showing the message in mulitple lines is not supported though unfortunately, as  VSCode explicitly strips newline characters from the message text.